src/NuGet.Core/NuGet.Packaging/ContentModel/ManagedCodeConventions.cs

```diff
@@ -22,7 +21,7 @@ public class ManagedCodeConventions
 
         private static readonly ContentPropertyDefinition AnyProperty = new ContentPropertyDefinition(
             PropertyNames.AnyValue,
-            parser: (o, t) => o); // Identity parser, all strings are valid for any
+            parser: (o, t) => o.ToString()); // Identity parser, all strings are valid for any
```

Comment:
What does this comment mean?

```suggestion
            parser: (o, t) => o.ToString()); // Identity parser, all strings are valid for any <??>
```